msgpack/fallback.py

```diff
@@ -234,6 +234,12 @@ def __init__(self, file_like=None, read_size=0, use_list=True, raw=True,
 
         #: array of bytes fed.
         self._buffer = bytearray()
+        # Some very old pythons don't support `struct.unpack_from()` with a
+        # `bytearray`. So we wrap it in a `buffer()` there.
+        if sys.version_info < (2, 7, 6):
```

Comment:
Note that the issue occurs on OSX with python 2.7.10... apparently this fix will not apply to OSX's /usr/bin/python

```
~: /usr/bin/python -c 'import sys; print(sys.version_info)'
sys.version_info(major=2, minor=7, micro=10, releaselevel='final', serial=0)
```